### PR TITLE
Support for translator 3.15.0

### PR DESCRIPTION
**Pull request by elsaperelli**
This PR bumps the version of the CQL translator from `3.7.1` to `3.15.0`. Release notes for `3.15.0` can be found [here](https://github.com/cqframework/clinical_quality_language/releases/tag/v3.15.0).

Summary of changes:
- In CQL translator version `3.10.0` (release notes [here](https://github.com/cqframework/clinical_quality_language/releases/tag/v3.10.0)), JUnit was updated to JUnit 5. I found [this page](https://junit.org/junit5/docs/current/user-guide/#migrating-from-junit4) useful for the updates made in `FormatterResourceTest.java` and `TranslationResourceTest.java`.
    - Please make sure I didn't miss anything else.
- Due to upgrading JUnit I also had to upgrade jersey and jaxb-runtime.

### CODE REVIEW

**Review by hossenlopp — APPROVED** [review]
This looks good and tests out with retranslating measures.

File: pom.xml [hunk 1]
```diff
@@ -62,11 +69,10 @@
...
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>4.13.2</version>
-      <scope>test</scope>
-    </dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter</artifactId>
+			<scope>test</scope>
+		</dependency>
```
Comment (cmoesel) [inline pom.xml:75]: These elements appear to be indented a good bit more than they should (if you don't see it, make sure you don't have "Hide whitespace" checked). I know it's kind of nit-picky, but could you fix the indentation so it's consistent with the rest?

**Review by cmoesel — CHANGES_REQUESTED** [review]
Thanks, Elsa!

I made one nit-picky comment about indented XML. I also noticed that this is out of sync with master; could you please merge or rebase to get in sync?

I assume that we will also want a new release (which would be release 2.5.0), so we will need to update the release number from 2.4.0 to 2.5.0 in:
* `pom.xml`
* `Dockerfile`
* `README.md` (two places, and also reflect the translator version 3.15.0 in the table)

We can either do that in this PR too (so we're ready to release right away) or we can put the 2.5.0 version bump in a separate PR following this one. I don't have a strong preference, so whatever is easier from your perspective.

Also -- don't worry about the failed GitHub Action for the Docker publish; that's actually probably a good thing (since we probably shouldn't push to DockerHub from forks).

**Review by cmoesel — APPROVED** [review]
Thanks, Elsa! I tested this manually as well as through Docker and all functionality is working as expected. And the JUnit 5 tests looks great. Thanks!

**Review by hossenlopp — APPROVED** [review]

**Pull request merged**